services/user.py

```diff
@@ -0,0 +1,39 @@
+from db.models import User
+
+
+def create_user(username: str, password: str, email: str = None,
+                first_name: str = None, last_name: str = None):
+    user = User.objects.create_user(username=username,
+                                    password=password)
+    if email:
+        user.email = email
+    if first_name:
+        user.first_name = first_name
+    if last_name:
+        user.last_name = last_name
+    user.save()
+    return user
+
+
+def get_user(user_id: int):
+    return User.objects.get(id=user_id)
```

Comment:
I think it is a better way
```suggestion
    return get_user_model().objects.get(id=user_id)
```